Libraries/Text/Text.js

```diff
@@ -11,6 +11,8 @@
  */
 'use strict';
 
+const ColorPropType = require('ColorPropType');
+const processColor = require('processColor');
```

Comment:
Nit: Move this with other function requires (those that start with a lowercase letter)